apollo-router/src/plugins/coprocessor/mod.rs

```diff
@@ -1137,29 +1138,28 @@ where
         let code = control.get_http_status()?;
 
         let res = {
-            let graphql_response: crate::graphql::Response =
-                match co_processor_output.body.unwrap_or(serde_json::Value::Null) {
-                    serde_json::Value::String(s) => crate::graphql::Response::builder()
```

Comment:
This part is weird, I think it would be better to just have custom error saying you need to provide `body` if you return Break. Instead we just push null instead of `body` that result in parsing error.
But I decided to stop myself here, since I already went to deep into refactoring this code.